arduino-ide-extension/src/browser/dialogs/settings/settings-step-input.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from '@theia/core/shared/react';
 import classnames from 'classnames';
+import _ = require('lodash');
```

Comment:
IDE2 does not directly depend on `lodash`, so this code can technically break.

```
yarn why lodash
yarn why v1.22.18
[1/4] 🤔  Why do we have the module "lodash"...?
[2/4] 🚚  Initialising dependency graph...
warning Resolution field "@types/react@16.14.25" is incompatible with requested version "@types/react@^17"
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "lodash@4.17.21"
info Reasons this module exists
   - "_project_#style-dictionary" depends on it
   - Hoisted from "_project_#style-dictionary#lodash"
   - Hoisted from "_project_#arduino-ide-extension#typemoq#lodash"
   - Hoisted from "_project_#electron-app#@theia#debug#requestretry#lodash"
   - Hoisted from "_project_#@theia#cli#mocha#yargs-unparser#lodash"
   - Hoisted from "_project_#jsdom#request-promise-native#request-promise-core#lodash"
   - Hoisted from "_project_#@theia#cli#@theia#ffmpeg#@electron#get#global-tunnel-ng#lodash"
   - Hoisted from "_project_#lerna#@lerna#clean#@lerna#prompt#inquirer#lodash"
   - Hoisted from "_project_#lerna#@lerna#version#@lerna#conventional-commits#conventional-changelog-core#lodash"
   - Hoisted from "_project_#lerna#@lerna#version#@lerna#conventional-commits#conventional-changelog-core#conventional-changelog-writer#lodash"
   - Hoisted from "_project_#lerna#@lerna#version#@lerna#conventional-commits#conventional-changelog-core#conventional-commits-parser#lodash"
info Disk size without dependencies: "4.88MB"
info Disk size with unique dependencies: "4.88MB"
info Disk size with transitive dependencies: "4.88MB"
info Number of shared dependencies: 0
✨  Done in 1.09s.
```

Could you please check that importing the entire `lodash` increased the frontend size by almost 5MB or was it already required?

```
info Disk size without dependencies: "4.88MB"
```

Reply:
> or was it already required?

It was.

with lodash 11.767.954 bytes (11,8 MB on disk)
without lodash 11.698.043 bytes (11,7 MB on disk)

Reply:
lodash is not a deal breaker here, the motivation for the PR is the reacty stuff, I thought it was a direct dependency...thanks for catching this